src/L1GovernorMetadataBridge.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {IGovernor} from "openzeppelin/governance/Governor.sol";
+import {IWormhole} from "wormhole/interfaces/IWormhole.sol";
+
+contract L1GovernorMetadataBridge {
```

Comment:
Nit: might be nice to have a general natspec comment quickly describing the purpose of the contract.

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/8/commits/187ca6f32075fc097c14cd1a81072cf5b245203a